snet_cli/commands.py

```diff
@@ -504,3 +510,247 @@ def transact(self):
+            tar.add(path, path.name) if path == deepest_path else tar.add(path, path.relative_to(deepest_path.parent))
+            self._printout("path: {}".format(str(path)))
+
+        tar.close()
+        return f
+
+    def _get_network(self):
+        if "network" in self.args and self.args.network:
+            if "endpoint" in self.args and self.args.endpoint:
+                network = self.args.endpoint
+            else:
+                network = self.config["network.{}".format(self.args.network)]['default_eth_rpc_endpoint']
+            self.w3 = get_web3(network)
+        return self.w3.version.network
+
+    def init(self):
+        accept_all_defaults = self.args.y
+        init_args = {
+            "name": os.path.basename(os.getcwd()),
+            "model": "model/",
+            "organization": "",
+            "price": 0,
+            "endpoint": "",
+            "tags": [],
+            "metadata": {
+                "description": ""
+            }
+        }
+
+        if not accept_all_defaults:
+            self._printout("Please provide values to populate your service.json file\n")
+
+        if self.args.name:
+            init_args["name"] = self.args.name
+        elif not accept_all_defaults:
+            init_args["name"] = input('Choose a name for your Service: (default: "{}")\n'.format(init_args["name"])) or init_args["name"]
+
+        if self.args.model:
+            init_args["model"] = self.args.model
+        elif not accept_all_defaults:
+            init_args["model"] = input('Choose the path to your Service model directory: (default: "{}")\n'.format(init_args["model"])) or init_args["model"]
+
+        if self.args.organization:
+            init_args["organization"] = self.args.organization
+        elif not accept_all_defaults:
+            init_args["organization"] = input('Choose an Organization for your Service: (default: "{}")\n'.format(init_args["organization"])) or init_args["organization"]
+
+        if self.args.price:
+            init_args["price"] = self.args.price
+        elif not accept_all_defaults:
+            init_args["price"] = input('Choose a price in AGI to call your Service: (default: {})\n'.format(init_args["price"])) or init_args["price"]
+
+        if self.args.endpoint:
+            init_args["endpoint"] = self.args.endpoint
+        elif not accept_all_defaults:
+            init_args["endpoint"] = input('URL to call your Service Agent: (default: "{}")\n'.format(init_args["endpoint"])) or init_args["endpoint"]
+
+        if self.args.tags:
+            init_args["tags"] = self.args.tags
+        elif not accept_all_defaults:
+            tags_user_input = input("Input a list of tags for your Service: (default: {})\n".format(init_args["tags"])) or init_args["tags"]
+            if tags_user_input and " " in tags_user_input and "," in tags_user_input:
+                raise ValueError("Please either provide a list separated either by comma or space, not both")
```

Comment:
"Please either provide a list separated either by comma or space, not both"
fix your grammar